build/Dockerfile

```diff
@@ -1,12 +1,24 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.14 AS builder
```

Comment:
Shouldn't we use 1.13 here to match up with what's in [go.mod](https://github.com/openshift/custom-domains-operator/blob/0a4313a3e6749d9f4b51eb55281dcfa867b51197/go.mod#L3)?

Reply:
Go 1.13 is [unsupported](https://golang.org/doc/devel/release.html) now. Should we just go ahead and update the `go.mod` file to be 1.14 because IIUC that's just used to parse the `go.mod` file itself.

Reply:
Good question. At some point we would like to include the go version in the standardization. And right now, I think most of the operators are using 1.13, so that would be the easiest place to start. Like we're talking about starting with operator-sdk 0.16 for the same reason.

But if we are going to flip to a newer go version, why not 1.15?

Reply:
I will just switch it to 1.13.